geps/gep-1897.md

```diff
@@ -0,0 +1,171 @@
+a backend pod that has its own certificate”. TLS configuration can be a nebulous topic,
+so in order to drive resolution this GEP focuses only on this single piece of functionality.
+
+## Immediate Goals
+
+1. The solution must satisfy the following use case: the backend pod has its own
+certificate and the gateway implementation client needs to know how to connect to the
+backend pod. (Use case #4 in [Gateway API TLS Use Cases](#references))
+2. In terms of the Gateway API personas, only the application developer persona in this
+solution. The application developer should control the gateway to backend TLS settings,
+not the cluster operator, as requiring a cluster operator to manage certificate renewals
+and revocations would be extremely cumbersome.
+3. The solution should consider client certificate settings used in the TLS handshake **from
+Gateway to backend**, such as TLS versions and cipher suites. (Use case #5 in [Gateway API TLS Use Cases](#references))
+
+## Longer Term Goals
+
+These are worthy goals, but may need a different GEP for proper attention.
+
+1. TCPRoute use cases (completed by GA)
+2. Mutual TLS use cases
+3. Service mesh use cases
+
+## Non-Goals
+
+These are worthy goals, but will not be covered by this GEP.
+
+1. Changes to the existing mechanisms for edge or passthrough TLS termination
+2. Providing a mechanism to decorate multiple route instances
+3. TLSRoute use cases
+4. UDPRoute use cases 
+5. Termination of TLS for HTTP routing (#1 in [Gateway API TLS Use Cases](#references))
+6. HTTPS passthrough use cases (#2 in [Gateway API TLS Use Cases](#references))
+7. Termination of TLS for non-HTTP TCP streams (#3 in [Gateway API TLS Use Cases](#references))
+8. Controlling certificates used by more than one workload (#6 in [Gateway API TLS Use Cases](#references))
+9. Client certificate settings used in TLS **from external clients to the
+Listener** (#7 in [Gateway API TLS Use Cases](#references))
+10. Providing a mechanism for the cluster operator to override gateway to backend TLS settings.
+
+## Overview - what do we want to do?
+
+Given that the current ingress solution specifies **edge** TLS termination (from the client to
+the gateway), and how to handle **passthrough** TLS (from the client to the backend pod), this
+proposed ingress solution specifies TLS origination to the **backend** (from the gateway to the
+backend pod).  As mentioned, this solution satisfies the use case in which the backend pod
+has its own certificate and the gateway client needs to know how to connect to the backend pod.
+
+![image depicting TLS termination types](images/1897-TLStermtypes.png "TLS termination types")
+
+Gateway API is missing a mechanism for separately providing the details for the backend TLS handshake,
+including:
+* use of TLS
+* destination CA (certificate authority) or CA bundle
+* SANs for validating upstream service (server authentication)
```

Comment:
This is another area where I'm not sure the HttpRoute author should make decisions ( egress is a slightly different case that I believe is not in scope here ). 

Currently Spiffee (URL) and DNS SANs are the majority of SANs in use. There are use cases where the DNS SAN may not be the same as authority - but not clear that the HttpRoute author is the best persona for configuring this ( as opposed to the service owner ).

Reply:
Indeed, we're just trying to acknowledge what the problem is and if it's worth working on. That said, if you feel there's a way we can better capture this and other comments which aren't clearly actionable at this stage, we've used a `TODO` list inside the GEP elsewhere it may be reasonable to suggest TODO items that need to be resolved before we move out of `Provisional` for some of your concerns, do feel free to suggest adding that to this PR.

Reply:
Added to Open Questions.